bundles/FileExplorerBundle/public/js/file.js

```diff
@@ -11,11 +11,11 @@
  * @license    http://www.pimcore.org/license     GPLv3 and PCL
  */
 
-pimcore.registerNS("pimcore.settings.fileexplorer.file");
+pimcore.registerNS("pimcore.bundle.file_explorer.settings.file");
```

Comment:
@brusch how should we tackle namespaces like these? 
Should we keep the underscore or replace it with a dash or dot?

Reply:
@aashan10 @mattamon dash won't work (not allowed for js objects), I'd keep the underscore here.